checkout-v3/payments-only/redirect.md

```diff
@@ -89,11 +59,95 @@ Once the payer has completed the purchase, you can perform a `GET` towards the
+            deactivate SwedbankPay
+            Merchant -->>- Payer: Redirect payer to SwedbankPay purchase page.
+    activate SwedbankPay
+    activate Payer
+    Payer ->> Payer: Initiate Purchase step
+
+    deactivate Payer
+    SwedbankPay ->>+ Payer: Do purchase logic
+    Payer ->> SwedbankPay: Do purchase logic
+    deactivate Payer
+    deactivate SwedbankPay
+
+                    opt Payer perform purchase out of iFrame
+                    activate Payer
+                    Payer ->> Payer: Redirect to 3rd party
+                    Payer ->>+ 3rdParty: Redirect to 3rdPartyUrl URL
+                    deactivate Payer
+                    3rdParty -->>+ Payer: Redirect back to SwedbankPay
+                    deactivate 3rdParty
+                    Payer ->> Payer: Initiate Payment Menu
+                    Payer ->>+ SwedbankPay: Show Purchase UI page in iframe
+                    deactivate Payer
+                end
+
+                activate SwedbankPay
+                SwedbankPay -->> Payer: Purchase status
+                deactivate SwedbankPay
+
+            alt If Purchase is completed
+            activate Payer
+            Payer ->> Payer: Redirect back to CompleteUrl
+            Payer ->>+ Merchant: Check Purchase status
+            deactivate Payer
+            Merchant ->>+ SwedbankPay: GET <paymentorder.id>
+            deactivate Merchant
+            SwedbankPay ->>+ Merchant: Status: Paid
+            deactivate SwedbankPay
+            end
+
+activate Merchant
+Merchant -->>- Payer: Show Purchase complete
+         opt PaymentOrder Callback (if callbackUrls is set) ①
+                activate SwedbankPay
+                SwedbankPay ->> Merchant: POST Purchase Callback
+                deactivate SwedbankPay
+         end
+         end
+
+    rect rgba(81,43,43,0.1)
+        activate Merchant
+        note left of Payer: Capture
+        Merchant ->>+ SwedbankPay: rel:capture
+        deactivate Merchant
+        SwedbankPay -->>- Merchant: Capture status
+        note right of Merchant: Capture here only if the purchased<br/>goods don't require shipping.<br/>If shipping is required, perform capture<br/>after the goods have shipped.<br>Should only be used for <br>PaymentInstruments that support <br>Authorizations.
+        end
+```
+
+*   ① Read more about [callback][payments-callback] handling in the technical reference.
+
+{% include iterator.html prev_href="/checkout-v3/payments-only/create-checkout-request"
+                         prev_title="Create Request"
                          next_href="post-purchase"
                          next_title="Post Purchase" %}
 
 [abort-feature]: /checkout-v3/payments-only/features/core/abort
+[features]: /checkout-v3/payments-only/features/
+[frictionless]: /checkout-v3/payments-only/features/core/3d-secure-2
+[order-items]: /checkout-v3/payments-only/features/optional/order-items
 [sequence-diagram]: /checkout-v3/sequence-diagrams#payments-only-redirect
```

Comment:
**Nitpick, Minor**

Remark, Codacy and Reviewdog.
Unused definitions from line 147 to 152. (Abort-feature to sequence-diagram).